packages/ui/src/components/SymbiosisBarChart/index.tsx

```diff
@@ -0,0 +1,98 @@
+import * as React from "react";
+
+import { Chart } from "../Charts";
+import type { ChartConfig, ChartDataPoint } from "../Charts/types";
+import type { SymbiosisBarChartProps } from "./types";
+import { cn } from "../../utils/cn";
+
+const SymbiosisBarChart = ({
+  data,
+  config,
+  xAxisFormatter,
+  tooltipLabelFormatter,
+  className,
+  legendClassName,
+  tooltipClassName,
+}: SymbiosisBarChartProps) => {
+
```

Comment:
should we add error handling for missing data?

```
  if (!data || data.length === 0) {
    return <div>No data available</div>;
  }
  ```

Reply:
I really prefer to delegate this responsibility to the consumer since it allows for complete control of this error handling case. 
The consumer can stylise and customise everything.

I'll add a `return null` though when data are not present. 👍

Reply:
Maybe not after all 😅. We will handle it in the future with the use case @bratsos proposed